### PR TITLE
feat(apimversioned.v3): Introduced Microsoft API version selector

### PR DESCRIPTION
**Pull request by jaq316**
For issue #30

### CODE REVIEW

**Comment by jaq316** [comment]
For issue #30

**Comment by stephaneey** [comment]
Hello @jaq316 ,

Thanks, this looks all right as you keep the previous version as the default one. I don't have time right now to make a full regression test session but that should be all right.

Best Regards

File: apimversioned/v3/apimversioned.ps1 [hunk 1]
```diff
@@ -15,6 +15,7 @@ shared VNET
         $VersionHeader= Get-VstsInput -Name VersionHeader
 		$QueryParam = get-VstsInput -Name QueryParam
 		$VersionScheme = Get-VstsInput -Name scheme 
+		$MicrosoftApiManagementAPIVersion - Get-VstsInput -Name MicrosoftApiManagementAPIVersion
```
Comment (jaq316) [inline apimversioned/v3/apimversioned.ps1]: Typo! Should be
`$MicrosoftApiManagementAPIVersion = Get-VstsInput -Name MicrosoftApiManagementAPIVersion`

Comment (stephaneey) [reply apimversioned/v3/apimversioned.ps1]: Indeed :)

**Pull request merged**